### PR TITLE
Jaustin/applepayfix

### PR DESCRIPTION
**Pull request by JohnMAustin78**
- Corrected Xcode build config doc step missing
- Corrected AndroidManifest.xml doc step missing
- Updated plugin version to 1.0

### CODE REVIEW

File: doc/get-started.md [hunk 1]
```diff
@@ -35,8 +36,32 @@ flutter create in_app_payments_flutter_app
...
 started guide for more detailed instructions.
 
-
-## Step 2: Configure the In-App Payments SDK dependency
+## Step 2: Add the In-App Payments SDK to your iOS project
+
+To use the In-App Payments plugin on iOS devices, install **In-App Payments SDK for iOS** 
+to make it an available resource for the Flutter library. 
+
+1. Open your iOS project `Runner.xcodeproj` with **Xcode** and complete
```
Comment (hukid) [inline doc/get-started.md]: flutter plugin has In-App Payments SDK embedded, so this step is not needed.

Comment (JohnMAustin78) [reply doc/get-started.md]: Removed manual build step, left instruction to open project in Xcode.

**Comment by hukid** [comment]
can you also update the `setIOSCardEntryTheme` in references.md, it should below to Card Entry

**Comment by hukid** [comment]
We need remove `INTERAC` and `SQUARE_GIFT_CARD` from references doc

**Pull request merged**